js/Events/logInEvents.js

```diff
@@ -17,13 +17,10 @@ interviewApp.Events.EventsLogIn = function () {
             displayAlert(event.target, 'Please complete both fields!');
             return;
         }
-        // ////////////////////////////////////////////////////////////////
-        const xhttp = new XMLHttpRequest();
-        xhttp.onreadystatechange = function () {
-            if (this.readyState == 4 && this.status == 200) {
-                // Typical action to be performed when the document is ready:
-
-                const logInObj = JSON.parse(xhttp.responseText);
+        const xhttpLogIn = new XMLHttpRequest();
+        xhttpLogIn.onreadystatechange = function () {
+            if (this.readyState === 4 && this.status === 200) {
+                const logInObj = JSON.parse(xhttpLogIn.responseText);
```

Comment:
Just one thing to note: `JSON.parse` is a potentially unsafe operation, depending on the value that gets loaded, in that it may throw errors if the value is not valid JSON. Thus, it is safest to always wrap in a `try` / `catch` block, unless you are positive that it you want to throw the error for some other code down the pipe to catch.

Reply:
I will take care of that!